airflow/timetables/interval.py

```diff
@@ -249,14 +249,17 @@ def infer_manual_data_interval(self, *, run_after: DateTime) -> DataInterval:
 
 
 class CronTriggerTimetable(_CronMixin, Timetable):
-    """Timetable that schedules in a plan cron-like fashion.
+    """Another timetable which accepts a cron expression. It behaves more
+    compliant with standard cron than ``CronDataIntervalTimetable``.
```

Comment:
Please keep the first line of the docstring succinct. See [PEP 257](https://peps.python.org/pep-0257/) for a general docstring style guide.

Reply:
Thanks for your prompt review! Made it shorter by https://github.com/apache/airflow/pull/23662/commits/74723e9ef2faa92d5cb389238ef5f5850adbe179